### DISCUSSION TITLE
Rangeshifter update - Crashing on RunRS

### DISCUSSION THREAD

**Discussion by ellalewi547** [RangeShifter/RangeshiftR-tutorials#47] [General]
I have downloaded version 1.1.0 of Rangeshifter and when I run my script with the same parameters as before the R session crashes. Before I updated to the new version I wasn't  experiencing any issues. Everything runs fine and the validateRSparams doesn't bring up any issues. I am using r version 4.3.1 currently. From an initial assessment do you know why this may be happening, is there anything  I can do to trouble shoot the issue?

Thank you, Ella

**Comment by JetteReeg** [comment]
Hey Ella, I am also currently encounter a similar issue and already looking into it. I will keep you updated.

**Reply by JetteReeg** [reply]
Could you please check whether the CELLSIZE parameter in the header of your raster files is an integer value or double/float? In my case, R crashes as soon as it is a double/float. If I manually delete the decimal digits (e.g. 10.000 -> 10) I had no issues.
 (I have solved a different bug regarding this a while ago, but now it seem to pop up again).

**Reply by ellalewi547** [reply]
Hi thank you for getting back to me so soon. Do I find this by looking at the Imported Landscape object and under resolution it says the type, if so it is double. How would I manually change this to an integer if I am importing my patch and land cover layers as asc. files?
Thank you

**Reply by JetteReeg** [reply]
You can open .asc files e.g. with an editor program such as notepad. Once openend, check whether the number behind cellsize has decimal digits. You can manually delete the decimal digits, save the file and test it again.

**Reply by ellalewi547** [reply]
That has worked thank you!

**Comment by JetteReeg** [comment]
Hi Ella, I was able to fix the issue. Now decimal digits are again allowed (or better the case is accounted for in RangeShiftR). A new installation of the RangeShiftR package should solve the issue. However, please wait until tomorrow. I still need to update the version number and create a new tag.
(I also saw the other issues, I hope to have a look at them tomorrow, but cannot promise....)

**Reply by ellalewi547** [reply]
That's great thank you!
